### DISCUSSION TITLE
Problems with remote bluetooth keyboard

### DISCUSSION THREAD

**Discussion by CariocoSuzuki** [osmandapp/OsmAnd#18723] [General]
I have installed a bluetooth remote keypad for my motorbike, Android 12 on DooGee S100 and sometimes and for no apparent reason when I send it map scrolling, up, down, right or left arrows, it works correctly and sometimes not.
To recover its correct functioning I have to change the navigation direction and for no apparent reason it works correctly again.
Any suggestions?

**Comment by DmitryAlexei** [comment]
What version of OsmAnd do you use? Similar issues have been fixed already. You can try beta from Google Play and see if it works for you.

**Comment by CariocoSuzuki** [comment]
I was working with version 4.5, following the forum's instructions I have installed version 4.6 Beta and the problem is apparently solved.
I think this may be enlightening. I use a DooGee S100 phone with Android 12
